templates/Test.cpp.ejs

```diff
@@ -12,6 +13,19 @@
 namespace PlayFabUnit
 {
 
+void AutoGen<%- api.name %>Tests::Log(std::stringstream& ss)
+{
+    TestApp::LogPut(ss.str().c_str());
+    ss.str(std::string());
+    ss.clear();
```

Comment:
whats the idea here? is the stringstream reused later?  strange semantics that a log function would clear the stream.

Reply:
ya otherwise i have to declare new vars or other oddness.

Reply:
can't you just call Log once at the end?

Reply:
perhaps, but LogPut() adds end line and expects lines to be sent, so i'd have to add << std::endl.   Seems a bit safer to log each line as it happens in case things crash, etc anyway